README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
@bkeepers We understand that, and there is 100s of other CoCs on github that have more egregious terms than this one, and they're ignored. The difference of course is that this one made it to the official github blog. Github the company is publicly promoting this CoC, and  recommending it for everyone's repo. 

> If your project doesn't already have a code of conduct, then we encourage you to check out the Open Code of Conduct and consider if your community can commit to upholding it. Read more about it on the TODO Group blog.

Since most projects care more about software development and less about politics, this will be the default one to copy/paste in. 

Microsoft making Internet Explorer the default browser in Windows circa 1999 didn't seem like a "big deal" prima facie. 

The attention brought in some trolls, and that sucks, but there's lots of us trying to help make this a reasonable CoC that's able to universally protect all kinds of people from harassment and discrimination and and let us get back to making awesome software together. :)

Thank you for working with us on this.